ext/hal/atmel/asf/sam0/include/samd20/sam.h

```diff
@@ -0,0 +1,70 @@
+/**

```

Comment:
This file is not required, we could remove it from commit -> less clutter. Other existing imports from Atmel ASF do not include this file.